garage/tf/algos/td3.py

```diff
@@ -0,0 +1,252 @@
+"""
+This module implements a TD3 model.
+
+TD3, or Twin Delayed Deep Deterministic Policy Gradient, uses actor-critic
+method to optimize the policy and reward prediction. Notably, it uses the
+minimum value of two critics instead of one to limit overestimation.
+"""
+
+import numpy as np
+import tensorflow as tf
+import tensorflow.contrib as tc
```

Comment:
please ensure that these are not deprecated on TF2.0. I think `tf.contrib` is getting removed

Reply:
Removed.